collection-scripts/gather_network_logs

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+BASE_COLLECTION_PATH="must-gather"
+NETWORK_LOG_PATH="${BASE_COLLECTION_PATH}/network_logs/"
+
+mkdir -p ${NETWORK_LOG_PATH}/
+
+WORKERS=$(oc get nodes --no-headers -o custom-columns=":metadata.name")
+
+for WORKER in $WORKERS; do
+    SDN_POD=$(oc -n openshift-sdn get pods --no-headers -o custom-columns=":metadata.name" --field-selector spec.nodeName=$WORKER -l app=sdn)
+    oc -n openshift-sdn exec $SDN_POD -- bash -c "iptables-save -c" > ${NETWORK_LOG_PATH}/${WORKER}_iptables
+    oc -n openshift-sdn exec $SDN_POD -- bash -c "ovs-vsctl show" > ${NETWORK_LOG_PATH}/${WORKER}_ovs_dump
+    oc -n openshift-sdn logs $SDN_POD > ${NETWORK_LOG_PATH}/${SDN_POD}_sdn_logs
+done
+
+MASTERS=$(oc get nodes --no-headers -o custom-columns=":metadata.name" -l node-role.kubernetes.io/master)
+
+for MASTER in $MASTERS; do
+    CONTROLLER_POD=$(oc -n openshift-sdn get pods --no-headers -o custom-columns=":metadata.name" --field-selector spec.nodeName=$MASTER -l app=sdn-controller)
+    oc -n openshift-sdn logs $CONTROLLER_POD > ${NETWORK_LOG_PATH}/${CONTROLLER_POD}_controller_logs
+done
```

Comment:
I believe this is already captured because of the operator<>namespace<>pod<>container<>log relationship when you as for the networking operator.

Reply:
Do you have a sample output from must-gather where we can check?

Reply:
https://gcsweb-ci.svc.ci.openshift.org/gcs/origin-ci-test/pr-logs/pull/operator-framework_operator-lifecycle-manager/889/pull-ci-operator-framework-operator-lifecycle-manager-master-e2e-aws/2148/artifacts/e2e-aws/must-gather/

Reply:
Can we resolve this comment?